diff -upr mate-themes-1.1.0.orig/desktop-themes/aldabra/Makefile.am mate-themes-1.1.0/desktop-themes/aldabra/Makefile.am
--- mate-themes-1.1.0.orig/desktop-themes/Aldabra/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/desktop-themes/Aldabra/Makefile.am	2012-02-12 16:39:24.000000000 +0100
@@ -20,5 +20,5 @@ CLEANFILES = index.theme
 
 index.theme: index.theme.in
 	$(AM_V_GEN)sed \
-		-e s!\@BACKGROUND_DIR\@!$(datadir)/pixmaps/backgrounds/mate/abstract! \
+		-e s!\@BACKGROUND_DIR\@!$(datadir)/backgrounds/mate/abstract! \
 		< $< > $@
diff -upr mate-themes-1.1.0.orig/desktop-themes/Clearlooks/Makefile.am mate-themes-1.1.0/desktop-themes/Clearlooks/Makefile.am
--- mate-themes-1.1.0.orig/desktop-themes/Clearlooks/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/desktop-themes/Clearlooks/Makefile.am	2012-02-12 16:46:02.000000000 +0100
@@ -3,7 +3,7 @@ THEME_IN_FILES=index.theme.in
  
 %.theme:   %.theme.in $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; $(INTLTOOL_MERGE) $(top_srcdir)/po $< $@ -d -u -c $(top_builddir)/po/.intltool-merge-cache
 
-themedir = $(datadir)/themes/$(THEME_NAME)
+themedir = $(datadir)/themes/MateClearlooks
 
 theme_DATA = index.theme
 
diff -upr mate-themes-1.1.0.orig/desktop-themes/ClearlooksTest/Makefile.am mate-themes-1.1.0/desktop-themes/ClearlooksTest/Makefile.am
--- mate-themes-1.1.0.orig/desktop-themes/ClearlooksTest/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/desktop-themes/ClearlooksTest/Makefile.am	2012-02-12 17:21:53.000000000 +0100
@@ -3,7 +3,7 @@ THEME_IN_FILES=index.theme.in
  
 %.theme:   %.theme.in $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; $(INTLTOOL_MERGE) $(top_srcdir)/po $< $@ -d -u -c $(top_builddir)/po/.intltool-merge-cache
 
-themedir = $(datadir)/themes/$(THEME_NAME)
+themedir = $(datadir)/themes/MateClearlooksTest
 
 theme_DATA = index.theme
 
diff -upr mate-themes-1.1.0.orig/desktop-themes/Crux/Makefile.am mate-themes-1.1.0/desktop-themes/Crux/Makefile.am
--- mate-themes-1.1.0.orig/desktop-themes/Crux/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/desktop-themes/Crux/Makefile.am	2012-02-12 17:17:26.000000000 +0100
@@ -3,7 +3,7 @@ THEME_IN_FILES=index.theme.in
  
 %.theme:   %.theme.in $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; $(INTLTOOL_MERGE) $(top_srcdir)/po $< $@ -d -u -c $(top_builddir)/po/.intltool-merge-cache
 
-themedir = $(datadir)/themes/$(THEME_NAME)
+themedir = $(datadir)/themes/MateCrux
 
 theme_DATA = index.theme
 
diff -upr mate-themes-1.1.0.orig/desktop-themes/Glider/Makefile.am mate-themes-1.1.0/desktop-themes/Glider/Makefile.am
--- mate-themes-1.1.0.orig/desktop-themes/Glider/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/desktop-themes/Glider/Makefile.am	2012-02-12 17:20:38.000000000 +0100
@@ -3,7 +3,7 @@ THEME_IN_FILES=index.theme.in
  
 %.theme:   %.theme.in $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; $(INTLTOOL_MERGE) $(top_srcdir)/po $< $@ -d -u -c $(top_builddir)/po/.intltool-merge-cache
 
-themedir = $(datadir)/themes/$(THEME_NAME)
+themedir = $(datadir)/themes/MateGlider
 
 theme_DATA = index.theme
 
diff -upr mate-themes-1.1.0.orig/desktop-themes/Glossy/Makefile.am mate-themes-1.1.0/desktop-themes/Glossy/Makefile.am
--- mate-themes-1.1.0.orig/desktop-themes/Glossy/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/desktop-themes/Glossy/Makefile.am	2012-02-12 17:20:25.000000000 +0100
@@ -3,7 +3,7 @@ THEME_IN_FILES=index.theme.in
  
 %.theme:   %.theme.in $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; $(INTLTOOL_MERGE) $(top_srcdir)/po $< $@ -d -u -c $(top_builddir)/po/.intltool-merge-cache
 
-themedir = $(datadir)/themes/$(THEME_NAME)
+themedir = $(datadir)/themes/MateGlossy
 
 theme_DATA = index.theme
 
diff -upr mate-themes-1.1.0.orig/desktop-themes/HighContrast/Makefile.am mate-themes-1.1.0/desktop-themes/HighContrast/Makefile.am
--- mate-themes-1.1.0.orig/desktop-themes/HighContrast/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/desktop-themes/HighContrast/Makefile.am	2012-02-12 17:20:07.000000000 +0100
@@ -3,7 +3,7 @@ THEME_IN_FILES=index.theme.in
 
 @INTLTOOL_THEME_RULE@ 
 
-themedir = $(datadir)/themes/$(THEME_NAME)
+themedir = $(datadir)/themes/MateHighContrast
 
 theme_DATA = index.theme
 
diff -upr mate-themes-1.1.0.orig/desktop-themes/HighContrastInverse/Makefile.am mate-themes-1.1.0/desktop-themes/HighContrastInverse/Makefile.am
--- mate-themes-1.1.0.orig/desktop-themes/HighContrastInverse/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/desktop-themes/HighContrastInverse/Makefile.am	2012-02-12 18:19:36.000000000 +0100
@@ -3,7 +3,7 @@ THEME_IN_FILES=index.theme.in
 
 @INTLTOOL_THEME_RULE@ 
 
-themedir = $(datadir)/themes/$(THEME_NAME)
+themedir = $(datadir)/themes/MateHighContrastInverse
 
 theme_DATA = index.theme
 
diff -upr mate-themes-1.1.0.orig/desktop-themes/HighContrastLargePrint/Makefile.am mate-themes-1.1.0/desktop-themes/HighContrastLargePrint/Makefile.am
--- mate-themes-1.1.0.orig/desktop-themes/HighContrastLargePrint/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/desktop-themes/HighContrastLargePrint/Makefile.am	2012-02-12 17:19:39.000000000 +0100
@@ -3,7 +3,7 @@ THEME_IN_FILES=index.theme.in
 
 @INTLTOOL_THEME_RULE@ 
 
-themedir = $(datadir)/themes/$(THEME_NAME)
+themedir = $(datadir)/themes/MateHighContrastLargePrint
 
 theme_DATA = index.theme
 
diff -upr mate-themes-1.1.0.orig/desktop-themes/HighContrastLargePrintInverse/Makefile.am mate-themes-1.1.0/desktop-themes/HighContrastLargePrintInverse/Makefile.am
--- mate-themes-1.1.0.orig/desktop-themes/HighContrastLargePrintInverse/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/desktop-themes/HighContrastLargePrintInverse/Makefile.am	2012-02-12 17:19:21.000000000 +0100
@@ -3,7 +3,7 @@ THEME_IN_FILES=index.theme.in
 
 @INTLTOOL_THEME_RULE@ 
 
-themedir = $(datadir)/themes/$(THEME_NAME)
+themedir = $(datadir)/themes/MateHighContrastLargePrintInverse
 
 theme_DATA = index.theme
 
diff -upr mate-themes-1.1.0.orig/desktop-themes/LargePrint/Makefile.am mate-themes-1.1.0/desktop-themes/LargePrint/Makefile.am
--- mate-themes-1.1.0.orig/desktop-themes/LargePrint/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/desktop-themes/LargePrint/Makefile.am	2012-02-12 17:19:07.000000000 +0100
@@ -3,7 +3,7 @@ THEME_IN_FILES=index.theme.in
 
 @INTLTOOL_THEME_RULE@ 
 
-themedir = $(datadir)/themes/$(THEME_NAME)
+themedir = $(datadir)/themes/MateLargePrint
 
 theme_DATA = index.theme
 
diff -upr mate-themes-1.1.0.orig/desktop-themes/LowContrast/Makefile.am mate-themes-1.1.0/desktop-themes/LowContrast/Makefile.am
--- mate-themes-1.1.0.orig/desktop-themes/LowContrast/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/desktop-themes/LowContrast/Makefile.am	2012-02-12 17:18:51.000000000 +0100
@@ -3,7 +3,7 @@ THEME_IN_FILES=index.theme.in
 
 @INTLTOOL_THEME_RULE@ 
 
-themedir = $(datadir)/themes/$(THEME_NAME)
+themedir = $(datadir)/themes/MateLowContrast
 
 theme_DATA = index.theme
 
diff -upr mate-themes-1.1.0.orig/desktop-themes/LowContrastLargePrint/Makefile.am mate-themes-1.1.0/desktop-themes/LowContrastLargePrint/Makefile.am
--- mate-themes-1.1.0.orig/desktop-themes/LowContrastLargePrint/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/desktop-themes/LowContrastLargePrint/Makefile.am	2012-02-12 17:18:27.000000000 +0100
@@ -3,7 +3,7 @@ THEME_IN_FILES=index.theme.in
 
 @INTLTOOL_THEME_RULE@ 
 
-themedir = $(datadir)/themes/$(THEME_NAME)
+themedir = $(datadir)/themes/MateLowContrastLargePrint
 
 theme_DATA = index.theme
 
diff -upr mate-themes-1.1.0.orig/desktop-themes/Mist/Makefile.am mate-themes-1.1.0/desktop-themes/Mist/Makefile.am
--- mate-themes-1.1.0.orig/desktop-themes/Mist/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/desktop-themes/Mist/Makefile.am	2012-02-12 17:17:53.000000000 +0100
@@ -3,7 +3,7 @@ THEME_IN_FILES=index.theme.in
  
 %.theme:   %.theme.in $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; $(INTLTOOL_MERGE) $(top_srcdir)/po $< $@ -d -u -c $(top_builddir)/po/.intltool-merge-cache
 
-themedir = $(datadir)/themes/$(THEME_NAME)
+themedir = $(datadir)/themes/MateMist
 
 theme_DATA = index.theme
  
diff -upr mate-themes-1.1.0.orig/gtk-themes/ClearlooksClassic/Makefile.am mate-themes-1.1.0/gtk-themes/ClearlooksClassic/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/ClearlooksClassic/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/ClearlooksClassic/Makefile.am	2012-02-12 16:52:15.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/themes/ClearlooksClassic/gtk-2.0
+themedir = $(datadir)/themes/MateClearlooksClassic/gtk-2.0
 
 theme_DATA = gtkrc
 
diff -upr mate-themes-1.1.0.orig/gtk-themes/ClearlooksTest/Makefile.am mate-themes-1.1.0/gtk-themes/ClearlooksTest/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/ClearlooksTest/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/ClearlooksTest/Makefile.am	2012-02-12 16:52:31.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/themes/ClearlooksTest/gtk-2.0
+themedir = $(datadir)/themes/MateClearlooksTest/gtk-2.0
 
 theme_DATA = gtkrc
 
diff -upr mate-themes-1.1.0.orig/gtk-themes/Glider/Makefile.am mate-themes-1.1.0/gtk-themes/Glider/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/Glider/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/Glider/Makefile.am	2012-02-12 16:52:44.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/themes/Glider/gtk-2.0
+themedir = $(datadir)/themes/MateGlider/gtk-2.0
 
 theme_DATA = gtkrc
 
diff -upr mate-themes-1.1.0.orig/gtk-themes/Glossy/Makefile.am mate-themes-1.1.0/gtk-themes/Glossy/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/Glossy/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/Glossy/Makefile.am	2012-02-12 16:52:58.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/themes/Glossy/gtk-2.0
+themedir = $(datadir)/themes/MateGlossy/gtk-2.0
 
 theme_DATA = gtkrc
 
diff -upr mate-themes-1.1.0.orig/gtk-themes/HighContrast/Makefile.am mate-themes-1.1.0/gtk-themes/HighContrast/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/HighContrast/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/HighContrast/Makefile.am	2012-02-12 16:53:12.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/themes/HighContrast/gtk-2.0
+themedir = $(datadir)/themes/MateHighContrast/gtk-2.0
 
 theme_DATA = gtkrc
 
diff -upr mate-themes-1.1.0.orig/gtk-themes/HighContrastInverse/Makefile.am mate-themes-1.1.0/gtk-themes/HighContrastInverse/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/HighContrastInverse/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/HighContrastInverse/Makefile.am	2012-02-12 16:53:25.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/themes/HighContrastInverse/gtk-2.0
+themedir = $(datadir)/themes/MateHighContrastInverse/gtk-2.0
 
 theme_DATA = gtkrc
 
diff -upr mate-themes-1.1.0.orig/gtk-themes/HighContrastLargePrint/Makefile.am mate-themes-1.1.0/gtk-themes/HighContrastLargePrint/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/HighContrastLargePrint/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/HighContrastLargePrint/Makefile.am	2012-02-12 16:53:41.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/themes/HighContrastLargePrint/gtk-2.0
+themedir = $(datadir)/themes/MateHighContrastLargePrint/gtk-2.0
 
 theme_DATA = gtkrc
 
diff -upr mate-themes-1.1.0.orig/gtk-themes/HighContrastLargePrint/pixmaps/Makefile.am mate-themes-1.1.0/gtk-themes/HighContrastLargePrint/pixmaps/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/HighContrastLargePrint/pixmaps/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/HighContrastLargePrint/pixmaps/Makefile.am	2012-02-12 16:53:55.000000000 +0100
@@ -1,4 +1,4 @@
-pixmapdir=	$(datadir)/themes/HighContrastLargePrint/pixmaps
+pixmapdir=	$(datadir)/themes/MateHighContrastLargePrint/pixmaps
 
 EXTRA_DIST=	$(pixmap_DATA)
 
diff -upr mate-themes-1.1.0.orig/gtk-themes/HighContrastLargePrintInverse/Makefile.am mate-themes-1.1.0/gtk-themes/HighContrastLargePrintInverse/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/HighContrastLargePrintInverse/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/HighContrastLargePrintInverse/Makefile.am	2012-02-12 16:54:11.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/themes/HighContrastLargePrintInverse/gtk-2.0
+themedir = $(datadir)/themes/MateHighContrastLargePrintInverse/gtk-2.0
 
 theme_DATA = gtkrc
 
diff -upr mate-themes-1.1.0.orig/gtk-themes/HighContrastLargePrintInverse/pixmaps/Makefile.am mate-themes-1.1.0/gtk-themes/HighContrastLargePrintInverse/pixmaps/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/HighContrastLargePrintInverse/pixmaps/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/HighContrastLargePrintInverse/pixmaps/Makefile.am	2012-02-12 16:54:21.000000000 +0100
@@ -1,4 +1,4 @@
-pixmapdir=	$(datadir)/themes/HighContrastLargePrintInverse/pixmaps
+pixmapdir=	$(datadir)/themes/MateHighContrastLargePrintInverse/pixmaps
 
 EXTRA_DIST=	$(pixmap_DATA)
 
diff -upr mate-themes-1.1.0.orig/gtk-themes/Inverted/Makefile.am mate-themes-1.1.0/gtk-themes/Inverted/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/Inverted/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/Inverted/Makefile.am	2012-02-12 16:54:37.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/themes/Inverted/gtk-2.0
+themedir = $(datadir)/themes/MateInverted/gtk-2.0
 
 theme_DATA = gtkrc
 
diff -upr mate-themes-1.1.0.orig/gtk-themes/LargePrint/Makefile.am mate-themes-1.1.0/gtk-themes/LargePrint/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/LargePrint/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/LargePrint/Makefile.am	2012-02-12 16:54:50.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/themes/LargePrint/gtk-2.0
+themedir = $(datadir)/themes/MateLargePrint/gtk-2.0
 
 theme_DATA = gtkrc
 
diff -upr mate-themes-1.1.0.orig/gtk-themes/LowContrast/Makefile.am mate-themes-1.1.0/gtk-themes/LowContrast/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/LowContrast/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/LowContrast/Makefile.am	2012-02-12 16:55:03.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/themes/LowContrast/gtk-2.0
+themedir = $(datadir)/themes/MateLowContrast/gtk-2.0
 
 theme_DATA = gtkrc
 
diff -upr mate-themes-1.1.0.orig/gtk-themes/LowContrastLargePrint/Makefile.am mate-themes-1.1.0/gtk-themes/LowContrastLargePrint/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/LowContrastLargePrint/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/LowContrastLargePrint/Makefile.am	2012-02-12 16:55:17.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/themes/LowContrastLargePrint/gtk-2.0
+themedir = $(datadir)/themes/MateLowContrastLargePrint/gtk-2.0
 
 theme_DATA = gtkrc
 
diff -upr mate-themes-1.1.0.orig/gtk-themes/LowContrastLargePrint/pixmaps/Makefile.am mate-themes-1.1.0/gtk-themes/LowContrastLargePrint/pixmaps/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/LowContrastLargePrint/pixmaps/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/LowContrastLargePrint/pixmaps/Makefile.am	2012-02-12 16:55:29.000000000 +0100
@@ -1,4 +1,4 @@
-pixmapdir=      $(datadir)/themes/LowContrastLargePrint/pixmaps
+pixmapdir=      $(datadir)/themes/MateLowContrastLargePrint/pixmaps
 
 EXTRA_DIST=	$(pixmap_DATA)
 
diff -upr mate-themes-1.1.0.orig/gtk-themes/Simple/Makefile.am mate-themes-1.1.0/gtk-themes/Simple/Makefile.am
--- mate-themes-1.1.0.orig/gtk-themes/Simple/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/gtk-themes/Simple/Makefile.am	2012-02-12 16:55:45.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/themes/Simple/gtk-2.0
+themedir = $(datadir)/themes/MateSimple/gtk-2.0
 
 theme_DATA = gtkrc
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Crux/16x16/actions/Makefile.am mate-themes-1.1.0/icon-themes/Crux/16x16/actions/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Crux/16x16/actions/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Crux/16x16/actions/Makefile.am	2012-02-12 16:32:14.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Crux
+themedir = $(datadir)/icons/MateCrux
 size = 16x16
 context = actions
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Crux/16x16/places/Makefile.am mate-themes-1.1.0/icon-themes/Crux/16x16/places/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Crux/16x16/places/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Crux/16x16/places/Makefile.am	2012-02-12 16:32:28.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Crux
+themedir = $(datadir)/icons/MateCrux
 size = 16x16
 context = places
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Crux/16x16/status/Makefile.am mate-themes-1.1.0/icon-themes/Crux/16x16/status/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Crux/16x16/status/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Crux/16x16/status/Makefile.am	2012-02-12 16:32:42.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Crux
+themedir = $(datadir)/icons/MateCrux
 size = 16x16
 context = status
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Crux/22x22/actions/Makefile.am mate-themes-1.1.0/icon-themes/Crux/22x22/actions/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Crux/22x22/actions/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Crux/22x22/actions/Makefile.am	2012-02-12 16:33:03.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Crux
+themedir = $(datadir)/icons/MateCrux
 size = 22x22
 context = actions
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Crux/22x22/places/Makefile.am mate-themes-1.1.0/icon-themes/Crux/22x22/places/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Crux/22x22/places/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Crux/22x22/places/Makefile.am	2012-02-12 16:33:15.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Crux
+themedir = $(datadir)/icons/MateCrux
 size = 22x22
 context = places
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Crux/22x22/status/Makefile.am mate-themes-1.1.0/icon-themes/Crux/22x22/status/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Crux/22x22/status/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Crux/22x22/status/Makefile.am	2012-02-12 16:33:28.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Crux
+themedir = $(datadir)/icons/MateCrux
 size = 22x22
 context = status
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Crux/24x24/actions/Makefile.am mate-themes-1.1.0/icon-themes/Crux/24x24/actions/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Crux/24x24/actions/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Crux/24x24/actions/Makefile.am	2012-02-12 16:33:44.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Crux
+themedir = $(datadir)/icons/MateCrux
 size = 24x24
 context = actions
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Crux/24x24/places/Makefile.am mate-themes-1.1.0/icon-themes/Crux/24x24/places/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Crux/24x24/places/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Crux/24x24/places/Makefile.am	2012-02-12 16:33:57.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Crux
+themedir = $(datadir)/icons/MateCrux
 size = 24x24
 context = places
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Crux/24x24/status/Makefile.am mate-themes-1.1.0/icon-themes/Crux/24x24/status/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Crux/24x24/status/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Crux/24x24/status/Makefile.am	2012-02-12 16:34:10.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Crux
+themedir = $(datadir)/icons/MateCrux
 size = 24x24
 context = status
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Crux/32x32/actions/Makefile.am mate-themes-1.1.0/icon-themes/Crux/32x32/actions/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Crux/32x32/actions/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Crux/32x32/actions/Makefile.am	2012-02-12 16:34:31.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Crux
+themedir = $(datadir)/icons/MateCrux
 size = 32x32
 context = actions
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Crux/32x32/places/Makefile.am mate-themes-1.1.0/icon-themes/Crux/32x32/places/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Crux/32x32/places/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Crux/32x32/places/Makefile.am	2012-02-12 16:34:43.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Crux
+themedir = $(datadir)/icons/MateCrux
 size = 32x32
 context = places
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Crux/32x32/status/Makefile.am mate-themes-1.1.0/icon-themes/Crux/32x32/status/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Crux/32x32/status/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Crux/32x32/status/Makefile.am	2012-02-12 16:34:54.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Crux
+themedir = $(datadir)/icons/MateCrux
 size = 32x32
 context = status
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Crux/scalable/actions/Makefile.am mate-themes-1.1.0/icon-themes/Crux/scalable/actions/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Crux/scalable/actions/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Crux/scalable/actions/Makefile.am	2012-02-12 16:35:20.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Crux
+themedir = $(datadir)/icons/MateCrux
 size = scalable
 context = actions
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Crux/scalable/places/Makefile.am mate-themes-1.1.0/icon-themes/Crux/scalable/places/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Crux/scalable/places/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Crux/scalable/places/Makefile.am	2012-02-12 16:35:33.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Crux
+themedir = $(datadir)/icons/MateCrux
 size = scalable
 context = places
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Crux/scalable/status/Makefile.am mate-themes-1.1.0/icon-themes/Crux/scalable/status/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Crux/scalable/status/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Crux/scalable/status/Makefile.am	2012-02-12 16:35:50.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Crux
+themedir = $(datadir)/icons/MateCrux
 size = scalable
 context = status
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/36x36/animations/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrint/36x36/animations/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/36x36/animations/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrint/36x36/animations/Makefile.am	2012-02-12 16:28:21.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrint
+themedir = $(datadir)/icons/MateHighContrastLargePrint
 size = 36x36
 context = animations
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/actions/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/actions/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/actions/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/actions/Makefile.am	2012-02-12 16:28:41.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrint
+themedir = $(datadir)/icons/MateHighContrastLargePrint
 size = 48x48
 context = actions
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/apps/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/apps/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/apps/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/apps/Makefile.am	2012-02-12 16:28:56.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrint
+themedir = $(datadir)/icons/MateHighContrastLargePrint
 size = 48x48
 context = apps
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/categories/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/categories/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/categories/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/categories/Makefile.am	2012-02-12 16:29:10.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrint
+themedir = $(datadir)/icons/MateHighContrastLargePrint
 size = 48x48
 context = categories
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/devices/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/devices/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/devices/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/devices/Makefile.am	2012-02-12 16:29:23.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrint
+themedir = $(datadir)/icons/MateHighContrastLargePrint
 size = 48x48
 context = devices
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/emblems/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/emblems/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/emblems/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/emblems/Makefile.am	2012-02-12 16:29:39.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrint
+themedir = $(datadir)/icons/MateHighContrastLargePrint
 size = 48x48
 context = emblems
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/mimetypes/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/mimetypes/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/mimetypes/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/mimetypes/Makefile.am	2012-02-12 16:29:52.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrint
+themedir = $(datadir)/icons/MateHighContrastLargePrint
 size = 48x48
 context = mimetypes
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/places/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/places/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/places/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/places/Makefile.am	2012-02-12 16:30:05.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrint
+themedir = $(datadir)/icons/MateHighContrastLargePrint
 size = 48x48
 context = places
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/status/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/status/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrint/48x48/status/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrint/48x48/status/Makefile.am	2012-02-12 16:30:20.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrint
+themedir = $(datadir)/icons/MateHighContrastLargePrint
 size = 48x48
 context = status
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/36x36/animations/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/36x36/animations/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/36x36/animations/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/36x36/animations/Makefile.am	2012-02-12 17:30:55.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrintInverse
+themedir = $(datadir)/icons/MateHighContrastLargePrintInverse
 size = 36x36
 context = animations
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/actions/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/actions/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/actions/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/actions/Makefile.am	2012-02-12 16:25:34.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrintInverse
+themedir = $(datadir)/icons/MateHighContrastLargePrintInverse
 size = 48x48
 context = actions
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/apps/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/apps/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/apps/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/apps/Makefile.am	2012-02-12 16:25:50.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrintInverse
+themedir = $(datadir)/icons/MateHighContrastLargePrintInverse
 size = 48x48
 context = apps
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/categories/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/categories/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/categories/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/categories/Makefile.am	2012-02-12 16:26:02.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrintInverse
+themedir = $(datadir)/icons/MateHighContrastLargePrintInverse
 size = 48x48
 context = categories
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/devices/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/devices/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/devices/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/devices/Makefile.am	2012-02-12 16:26:14.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrintInverse
+themedir = $(datadir)/icons/MateHighContrastLargePrintInverse
 size = 48x48
 context = devices
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/emblems/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/emblems/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/emblems/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/emblems/Makefile.am	2012-02-12 16:26:26.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrintInverse
+themedir = $(datadir)/icons/MateHighContrastLargePrintInverse
 size = 48x48
 context = emblems
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/mimetypes/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/mimetypes/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/mimetypes/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/mimetypes/Makefile.am	2012-02-12 16:26:39.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrintInverse
+themedir = $(datadir)/icons/MateHighContrastLargePrintInverse
 size = 48x48
 context = mimetypes
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/places/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/places/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/places/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/places/Makefile.am	2012-02-12 16:26:51.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrintInverse
+themedir = $(datadir)/icons/MateHighContrastLargePrintInverse
 size = 48x48
 context = places
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/status/Makefile.am mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/status/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrastLargePrintInverse/48x48/status/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrastLargePrintInverse/48x48/status/Makefile.am	2012-02-12 16:27:10.000000000 +0100
@@ -1,6 +1,6 @@
 NULL=
 
-themedir = $(datadir)/icons/HighContrastLargePrintInverse
+themedir = $(datadir)/icons/MateHighContrastLargePrintInverse
 size = 48x48
 context = status
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/48x48/animations/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/48x48/animations/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/48x48/animations/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/48x48/animations/Makefile.am	2012-02-12 16:19:12.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = 48x48
 context = animations
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/48x48/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/48x48/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/48x48/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/48x48/Makefile.am	2012-02-12 16:18:54.000000000 +0100
@@ -1,5 +1,5 @@
 
-themedir=	$(datadir)/icons/HighContrast-SVG/48x48
+themedir=	$(datadir)/icons/MateHighContrast-SVG/48x48
 
 SUBDIRS = animations
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/actions/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/actions/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/actions/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/actions/Makefile.am	2012-02-12 16:20:05.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = actions
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/actions-extra/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/actions-extra/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/actions-extra/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/actions-extra/Makefile.am	2012-02-12 16:20:18.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = actions
 
@@ -9,4 +9,4 @@ icon_DATA = \
 	history-view.svg
 
 EXTRA_DIST = \
-	$(icon_DATA)
\ No newline at end of file
+	$(icon_DATA)
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/apps/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/apps/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/apps/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/apps/Makefile.am	2012-02-12 16:20:30.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = apps
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/apps-extra/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/apps-extra/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/apps-extra/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/apps-extra/Makefile.am	2012-02-12 16:20:42.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = apps
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/categories/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/categories/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/categories/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/categories/Makefile.am	2012-02-12 16:20:56.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = categories
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/categories-extra/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/categories-extra/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/categories-extra/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/categories-extra/Makefile.am	2012-02-12 16:21:11.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = categories
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/devices/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/devices/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/devices/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/devices/Makefile.am	2012-02-12 16:21:24.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = devices
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/devices-extra/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/devices-extra/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/devices-extra/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/devices-extra/Makefile.am	2012-02-12 16:21:38.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = devices
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/emblems/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/emblems/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/emblems/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/emblems/Makefile.am	2012-02-12 16:21:52.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = emblems
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/emblems-extra/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/emblems-extra/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/emblems-extra/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/emblems-extra/Makefile.am	2012-02-12 16:22:04.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = emblems
 
@@ -7,4 +7,4 @@ icondir = $(themedir)/$(size)/$(context)
 icon_DATA =
 
 EXTRA_DIST = \
-	$(icon_DATA)
\ No newline at end of file
+	$(icon_DATA)
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/emotes/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/emotes/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/emotes/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/emotes/Makefile.am	2012-02-12 16:22:16.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = emotes
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/emotes-extra/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/emotes-extra/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/emotes-extra/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/emotes-extra/Makefile.am	2012-02-12 16:22:28.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = emotes
 
@@ -7,4 +7,4 @@ icondir = $(themedir)/$(size)/$(context)
 icon_DATA =
 
 EXTRA_DIST = \
-	$(icon_DATA)
\ No newline at end of file
+	$(icon_DATA)
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/Makefile.am	2012-02-12 16:19:48.000000000 +0100
@@ -1,5 +1,5 @@
 
-themedir=	$(datadir)/icons/HighContrast-SVG/scalable
+themedir=	$(datadir)/icons/MateHighContrast-SVG/scalable
 
 SUBDIRS = \
 	actions			\
@@ -19,4 +19,4 @@ SUBDIRS = \
 	emotes-extra		\
 	mimetypes-extra		\
 	places-extra		\
-	status-extra
\ No newline at end of file
+	status-extra
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/mimetypes/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/mimetypes/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/mimetypes/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/mimetypes/Makefile.am	2012-02-12 16:22:41.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = mimetypes
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/mimetypes-extra/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/mimetypes-extra/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/mimetypes-extra/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/mimetypes-extra/Makefile.am	2012-02-12 16:22:54.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = mimetypes
 
@@ -7,4 +7,4 @@ icondir = $(themedir)/$(size)/$(context)
 icon_DATA =
 
 EXTRA_DIST = \
-	$(icon_DATA)
\ No newline at end of file
+	$(icon_DATA)
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/places/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/places/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/places/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/places/Makefile.am	2012-02-12 16:23:06.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = places
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/places-extra/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/places-extra/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/places-extra/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/places-extra/Makefile.am	2012-02-12 16:23:20.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = places
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/status/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/status/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/status/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/status/Makefile.am	2012-02-12 16:23:35.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = status
 
diff -upr mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/status-extra/Makefile.am mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/status-extra/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/HighContrast-SVG/scalable/status-extra/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/HighContrast-SVG/scalable/status-extra/Makefile.am	2012-02-12 16:23:48.000000000 +0100
@@ -1,4 +1,4 @@
-themedir =  $(datadir)/icons/HighContrast-SVG
+themedir =  $(datadir)/icons/MateHighContrast-SVG
 size = scalable
 context = status
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/16x16/actions/Makefile.am mate-themes-1.1.0/icon-themes/Mist/16x16/actions/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/16x16/actions/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/16x16/actions/Makefile.am	2012-02-12 16:10:23.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 16x16
 context = actions
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/16x16/apps/Makefile.am mate-themes-1.1.0/icon-themes/Mist/16x16/apps/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/16x16/apps/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/16x16/apps/Makefile.am	2012-02-12 16:10:36.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 16x16
 context = apps
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/16x16/devices/Makefile.am mate-themes-1.1.0/icon-themes/Mist/16x16/devices/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/16x16/devices/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/16x16/devices/Makefile.am	2012-02-12 16:10:50.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 16x16
 context = devices
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/16x16/places/Makefile.am mate-themes-1.1.0/icon-themes/Mist/16x16/places/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/16x16/places/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/16x16/places/Makefile.am	2012-02-12 16:11:10.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 16x16
 context = places
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/16x16/status/Makefile.am mate-themes-1.1.0/icon-themes/Mist/16x16/status/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/16x16/status/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/16x16/status/Makefile.am	2012-02-12 16:11:41.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 16x16
 context = status
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/22x22/actions/Makefile.am mate-themes-1.1.0/icon-themes/Mist/22x22/actions/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/22x22/actions/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/22x22/actions/Makefile.am	2012-02-12 16:12:05.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 22x22
 context = actions
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/22x22/apps/Makefile.am mate-themes-1.1.0/icon-themes/Mist/22x22/apps/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/22x22/apps/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/22x22/apps/Makefile.am	2012-02-12 16:12:17.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 22x22
 context = apps
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/22x22/devices/Makefile.am mate-themes-1.1.0/icon-themes/Mist/22x22/devices/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/22x22/devices/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/22x22/devices/Makefile.am	2012-02-12 16:12:31.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 22x22
 context = devices
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/22x22/places/Makefile.am mate-themes-1.1.0/icon-themes/Mist/22x22/places/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/22x22/places/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/22x22/places/Makefile.am	2012-02-12 16:12:43.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 22x22
 context = places
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/22x22/status/Makefile.am mate-themes-1.1.0/icon-themes/Mist/22x22/status/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/22x22/status/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/22x22/status/Makefile.am	2012-02-12 16:12:56.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 22x22
 context = status
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/24x24/actions/Makefile.am mate-themes-1.1.0/icon-themes/Mist/24x24/actions/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/24x24/actions/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/24x24/actions/Makefile.am	2012-02-12 16:13:20.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 24x24
 context = actions
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/24x24/apps/Makefile.am mate-themes-1.1.0/icon-themes/Mist/24x24/apps/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/24x24/apps/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/24x24/apps/Makefile.am	2012-02-12 16:13:31.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 24x24
 context = apps
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/24x24/devices/Makefile.am mate-themes-1.1.0/icon-themes/Mist/24x24/devices/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/24x24/devices/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/24x24/devices/Makefile.am	2012-02-12 16:13:43.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 24x24
 context = devices
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/24x24/places/Makefile.am mate-themes-1.1.0/icon-themes/Mist/24x24/places/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/24x24/places/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/24x24/places/Makefile.am	2012-02-12 16:13:59.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 24x24
 context = places
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/24x24/status/Makefile.am mate-themes-1.1.0/icon-themes/Mist/24x24/status/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/24x24/status/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/24x24/status/Makefile.am	2012-02-12 16:14:12.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 24x24
 context = status
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/256x256/actions/Makefile.am mate-themes-1.1.0/icon-themes/Mist/256x256/actions/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/256x256/actions/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/256x256/actions/Makefile.am	2012-02-12 17:02:49.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 256x256
 context = actions
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/256x256/apps/Makefile.am mate-themes-1.1.0/icon-themes/Mist/256x256/apps/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/256x256/apps/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/256x256/apps/Makefile.am	2012-02-12 17:03:05.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 256x256
 context = apps
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/256x256/devices/Makefile.am mate-themes-1.1.0/icon-themes/Mist/256x256/devices/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/256x256/devices/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/256x256/devices/Makefile.am	2012-02-12 17:03:30.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 256x256
 context = devices
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/256x256/places/Makefile.am mate-themes-1.1.0/icon-themes/Mist/256x256/places/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/256x256/places/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/256x256/places/Makefile.am	2012-02-12 17:03:48.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 256x256
 context = places
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/256x256/status/Makefile.am mate-themes-1.1.0/icon-themes/Mist/256x256/status/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/256x256/status/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/256x256/status/Makefile.am	2012-02-12 17:04:05.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 256x256
 context = status
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/32x32/actions/Makefile.am mate-themes-1.1.0/icon-themes/Mist/32x32/actions/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/32x32/actions/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/32x32/actions/Makefile.am	2012-02-12 16:15:05.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 32x32
 context = actions
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/32x32/apps/Makefile.am mate-themes-1.1.0/icon-themes/Mist/32x32/apps/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/32x32/apps/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/32x32/apps/Makefile.am	2012-02-12 16:15:17.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 32x32
 context = apps
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/32x32/devices/Makefile.am mate-themes-1.1.0/icon-themes/Mist/32x32/devices/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/32x32/devices/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/32x32/devices/Makefile.am	2012-02-12 16:15:29.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 32x32
 context = devices
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/32x32/places/Makefile.am mate-themes-1.1.0/icon-themes/Mist/32x32/places/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/32x32/places/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/32x32/places/Makefile.am	2012-02-12 16:15:41.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 32x32
 context = places
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/32x32/status/Makefile.am mate-themes-1.1.0/icon-themes/Mist/32x32/status/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/32x32/status/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/32x32/status/Makefile.am	2012-02-12 16:15:55.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 32x32
 context = status
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/48x48/actions/Makefile.am mate-themes-1.1.0/icon-themes/Mist/48x48/actions/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/48x48/actions/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/48x48/actions/Makefile.am	2012-02-12 16:16:33.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 48x48
 context = actions
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/48x48/apps/Makefile.am mate-themes-1.1.0/icon-themes/Mist/48x48/apps/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/48x48/apps/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/48x48/apps/Makefile.am	2012-02-12 16:16:44.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 48x48
 context = apps
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/48x48/devices/Makefile.am mate-themes-1.1.0/icon-themes/Mist/48x48/devices/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/48x48/devices/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/48x48/devices/Makefile.am	2012-02-12 16:16:57.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 48x48
 context = devices
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/48x48/places/Makefile.am mate-themes-1.1.0/icon-themes/Mist/48x48/places/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/48x48/places/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/48x48/places/Makefile.am	2012-02-12 16:17:09.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 48x48
 context = places
 
diff -upr mate-themes-1.1.0.orig/icon-themes/Mist/48x48/status/Makefile.am mate-themes-1.1.0/icon-themes/Mist/48x48/status/Makefile.am
--- mate-themes-1.1.0.orig/icon-themes/Mist/48x48/status/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/icon-themes/Mist/48x48/status/Makefile.am	2012-02-12 16:17:23.000000000 +0100
@@ -1,4 +1,4 @@
-themedir = $(datadir)/icons/Mist
+themedir = $(datadir)/icons/MateMist
 size = 48x48
 context = status
 
Only in mate-themes-1.1.0.orig/marco-themes: Clearlooks
Only in mate-themes-1.1.0.orig/marco-themes: ClearlooksClassic
Only in mate-themes-1.1.0.orig/marco-themes: Glider
Only in mate-themes-1.1.0.orig/marco-themes: Glossy
Only in mate-themes-1.1.0.orig/marco-themes: Inverted
diff -upr mate-themes-1.1.0.orig/marco-themes/Makefile.am mate-themes-1.1.0/marco-themes/Makefile.am
--- mate-themes-1.1.0.orig/marco-themes/Makefile.am	2011-12-11 11:29:01.000000000 +0100
+++ mate-themes-1.1.0/marco-themes/Makefile.am	2012-02-12 17:48:41.000000000 +0100
@@ -1,10 +1,10 @@
 THEMES= \
-	Clearlooks \
-	ClearlooksClassic \
-	Glider \
-	Glossy \
-	Inverted \
-	Mist
+	MateClearlooks \
+	MateClearlooksClassic \
+	MateGlider \
+	MateGlossy \
+	MateInverted \
+	MateMist
 
 THEME_DIR=$(datadir)/themes
 THEME_SUBDIR=marco-1
Only in mate-themes-1.1.0/marco-themes: MateClearlooks
Only in mate-themes-1.1.0/marco-themes: MateClearlooksClassic
Only in mate-themes-1.1.0/marco-themes: MateGlider
Only in mate-themes-1.1.0/marco-themes: MateGlossy
Only in mate-themes-1.1.0/marco-themes: MateInverted
Only in mate-themes-1.1.0/marco-themes: MateMist
Only in mate-themes-1.1.0.orig/marco-themes: Mist
