--- libmate-2.7.2/schemas/desktop_mate_peripherals_mouse.schemas.in.default-cursor	2004-08-11 13:41:08.255401604 +0200
+++ libmate-2.7.2/schemas/desktop_mate_peripherals_mouse.schemas.in	2004-08-11 13:41:47.042682149 +0200
@@ -100,6 +100,7 @@
     <schema>
       <key>/schemas/desktop/mate/peripherals/mouse/cursor_theme</key>
       <applyto>/desktop/mate/peripherals/mouse/cursor_theme</applyto>
+      <default>dmz-aa</default>
       <owner>mate</owner>
       <type>string</type>
       <locale name="C">
@@ -118,7 +119,7 @@
       <applyto>/desktop/mate/peripherals/mouse/cursor_size</applyto>
       <owner>mate</owner>
       <type>int</type>
-      <default>18</default>
+      <default>24</default>
       <locale name="C">
 	<short>Cursor size</short>
 	<long>
