diff -Nur netspeed_applet-0.16-orig/src/netspeed.c netspeed_applet-0.16/src/netspeed.c
--- netspeed_applet-0.16-orig/src/netspeed.c	2010-06-05 16:02:11.000000000 +0200
+++ netspeed_applet-0.16/src/netspeed.c	2010-11-03 15:04:38.000000000 +0100
@@ -674,12 +674,12 @@
 
 	/* Refresh the values of the Infodialog */
 	if (applet->inbytes_text) {
-		inbytes = bytes_to_string((double)applet->devinfo.rx, FALSE, applet->show_bits);
+		inbytes = bytes_to_string((double)applet->devinfo.rx, FALSE, FALSE);
 		gtk_label_set_text(GTK_LABEL(applet->inbytes_text), inbytes);
 		g_free(inbytes);
 	}	
 	if (applet->outbytes_text) {
-		outbytes = bytes_to_string((double)applet->devinfo.tx, FALSE, applet->show_bits);
+		outbytes = bytes_to_string((double)applet->devinfo.tx, FALSE, FALSE);
 		gtk_label_set_text(GTK_LABEL(applet->outbytes_text), outbytes);
 		g_free(outbytes);
 	}
