diff -Nur netspeed_applet-0.16-orig/src/netspeed.c netspeed_applet-0.16/src/netspeed.c
--- netspeed_applet-0.16-orig/src/netspeed.c	2010-06-05 16:02:11.000000000 +0200
+++ netspeed_applet-0.16/src/netspeed.c	2010-07-15 11:35:54.000000000 +0200
@@ -362,14 +362,22 @@
 {
 	GtkIconTheme *icon_theme;
 	int i;
+	GdkPixbuf *pixbuf;
 	
 	icon_theme = gtk_icon_theme_get_default();
 
 	for (i = 0; i < 4; i++) {
 		if (applet->qual_pixbufs[i])
 			g_object_unref(applet->qual_pixbufs[i]);
-		applet->qual_pixbufs[i] = gtk_icon_theme_load_icon(icon_theme, 
+		pixbuf = gtk_icon_theme_load_icon(icon_theme, 
 			wireless_quality_icon[i], 24, 0, NULL);
+		if (pixbuf) {
+		  applet->qual_pixbufs[i] = gdk_pixbuf_copy(pixbuf);
+		  g_object_unref(pixbuf);
+		}
+		else {
+		  applet->qual_pixbufs[i] = NULL;
+		}
 	}
 }
 
