diff -upr mate-doc-utils-1.2.1-orig/bootstrap.make mate-doc-utils-1.2.1/bootstrap.make
--- mate-doc-utils-1.2.1-orig/bootstrap.make	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/bootstrap.make	2012-03-01 22:12:32.000000000 +0100
@@ -5,7 +5,7 @@ _db2omf  = $(top_srcdir)/xslt/docbook/om
 _rngdoc  = $(top_srcdir)/xslt/rngdoc/rngdoc.xsl
 _xsldoc  = $(top_srcdir)/xslt/xsldoc/xsldoc.xsl
 
-_malrng  = $(top_builddir)/rng/mallard/mallard.rng
+_malrng  = $(top_builddir)/rng/mallard/matemallard.rng
 
 _chunks  = $(top_srcdir)/xslt/docbook/utils/chunks.xsl
 _credits = $(top_srcdir)/xslt/docbook/utils/credits.xsl
diff -upr mate-doc-utils-1.2.1-orig/configure.in mate-doc-utils-1.2.1/configure.in
--- mate-doc-utils-1.2.1-orig/configure.in	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/configure.in	2012-03-02 01:23:27.000000000 +0100
@@ -67,7 +67,7 @@ AC_CONFIG_FILES([
 	tools/mate-doc-utils.pc
 	xslt/Makefile
 	xml2po/Makefile
-	xml2po/xml2po.pc
+	xml2po/matexml2po.pc
 	xml2po/xml2po/Makefile
 	xml2po/xml2po/modes/Makefile
 	xml2po/examples/Makefile
diff -upr mate-doc-utils-1.2.1-orig/doc/mate-doc-make/C/mate-doc-make.xml mate-doc-utils-1.2.1/doc/mate-doc-make/C/mate-doc-make.xml
--- mate-doc-utils-1.2.1-orig/doc/mate-doc-make/C/mate-doc-make.xml	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/doc/mate-doc-make/C/mate-doc-make.xml	2012-03-01 20:45:17.000000000 +0100
@@ -79,7 +79,7 @@
       <orderedlist>
         <listitem>
           <para>Translation is made much easier through the use of
-          <application>xml2po</application>, which generates standard
+          <application>matexml2po</application>, which generates standard
           <filename>.po</filename> files from docbook/xml.</para>
         </listitem>
 
diff -upr mate-doc-utils-1.2.1-orig/doc/mate-doc-make/de/de.po mate-doc-utils-1.2.1/doc/mate-doc-make/de/de.po
--- mate-doc-utils-1.2.1-orig/doc/mate-doc-make/de/de.po	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/doc/mate-doc-make/de/de.po	2012-03-01 20:45:45.000000000 +0100
@@ -202,11 +202,11 @@ msgstr ""
 
 #: mate-doc-make.xml:81(para)
 msgid ""
-"Translation is made much easier through the use of <application>xml2po</"
+"Translation is made much easier through the use of <application>matexml2po</"
 "application>, which generates standard <filename>.po</filename> files from "
 "docbook/xml."
 msgstr ""
-"Übersetzungen werden durch die Nutzung von <application>xml2po</application> "
+"Übersetzungen werden durch die Nutzung von <application>matexml2po</application> "
 "erheblich vereinfacht, welches standardisierte <filename>.po</filename>-"
 "Dateien aus DocBook/XML erzeugt."
 
diff -upr mate-doc-utils-1.2.1-orig/rng/mallard/Makefile.am mate-doc-utils-1.2.1/rng/mallard/Makefile.am
--- mate-doc-utils-1.2.1-orig/rng/mallard/Makefile.am	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/rng/mallard/Makefile.am	2012-03-01 21:38:51.000000000 +0100
@@ -1,4 +1,4 @@
 specdir = $(datadir)/xml/mallard/1.0/
-spec_DATA = mallard.rnc mallard.rng
+spec_DATA = matemallard.rnc matemallard.rng
 
 EXTRA_DIST = $(spec_DATA)
diff -upr mate-doc-utils-1.2.1-orig/tools/mate-doc-utils.make mate-doc-utils-1.2.1/tools/mate-doc-utils.make
--- mate-doc-utils-1.2.1-orig/tools/mate-doc-utils.make	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/tools/mate-doc-utils.make	2012-03-01 21:57:36.000000000 +0100
@@ -130,8 +130,8 @@ _DOC_ABS_SRCDIR = @abs_srcdir@
 ################################################################################
 ## Variables for Bootstrapping
 
-_xml2po ?= `which xml2po`
-_xml2po_mode = $(if $(DOC_ID),mallard,docbook)
+_matexml2po ?= `which matexml2po`
+_matexml2po_mode = $(if $(DOC_ID),mallard,docbook)
 
 _db2html ?= `$(PKG_CONFIG) --variable db2html mate-doc-utils`
 _db2omf  ?= `$(PKG_CONFIG) --variable db2omf mate-doc-utils`
@@ -327,16 +327,16 @@ $(_DOC_POFILES):
 	done; \
 	if ! test -f $@; then \
 	  echo "(cd $(dir $@) && \
-	    $(_xml2po) -m $(_xml2po_mode) -e $$docs > $(notdir $@).tmp && \
+	    $(_matexml2po) -m $(_matexml2po_mode) -e $$docs > $(notdir $@).tmp && \
 	    cp $(notdir $@).tmp $(notdir $@) && rm -f $(notdir $@).tmp)"; \
 	  (cd $(dir $@) && \
-	    $(_xml2po) -m $(_xml2po_mode) -e $$docs > $(notdir $@).tmp && \
+	    $(_matexml2po) -m $(_matexml2po_mode) -e $$docs > $(notdir $@).tmp && \
 	    cp $(notdir $@).tmp $(notdir $@) && rm -f $(notdir $@).tmp); \
 	else \
 	  echo "(cd $(dir $@) && \
-	    $(_xml2po) -m $(_xml2po_mode) -e -u $(notdir $@) $$docs)"; \
+	    $(_matexml2po) -m $(_matexml2po_mode) -e -u $(notdir $@) $$docs)"; \
 	  (cd $(dir $@) && \
-	    $(_xml2po) -m $(_xml2po_mode) -e -u $(notdir $@) $$docs); \
+	    $(_matexml2po) -m $(_matexml2po_mode) -e -u $(notdir $@) $$docs); \
 	fi
 
 $(_DOC_MOFILES): %.mo: %.po
@@ -347,7 +347,7 @@ $(_DOC_MOFILES): %.mo: %.po
 	msgfmt -o $@ $<
 
 # FIXME: fix the dependancy
-# FIXME: hook xml2po up
+# FIXME: hook matexml2po up
 $(_DOC_LC_DOCS) : $(_DOC_MOFILES)
 $(_DOC_LC_DOCS) : $(_DOC_C_DOCS)
 	if ! test -d $(dir $@); then mkdir $(dir $@); fi
@@ -355,7 +355,7 @@ $(_DOC_LC_DOCS) : $(_DOC_C_DOCS)
 	mo="$(dir $@)$(patsubst %/$(notdir $@),%,$@).mo"; \
 	if [ -f "$${mo}" ]; then mo="../$${mo}"; else mo="$(_DOC_ABS_SRCDIR)/$${mo}"; fi; \
 	(cd $(dir $@) && \
-	  $(_xml2po) -m $(_xml2po_mode) -e -t "$${mo}" \
+	  $(_matexml2po) -m $(_matexml2po_mode) -e -t "$${mo}" \
 	    "$${d}C/$(notdir $@)" > $(notdir $@).tmp && \
 	    cp $(notdir $@).tmp $(notdir $@) && rm -f $(notdir $@).tmp)
 
@@ -365,7 +365,7 @@ _DOC_POT = $(if $(DOC_MODULE),$(DOC_MODU
 .PHONY: pot
 pot: $(_DOC_POT)
 $(_DOC_POT): $(_DOC_C_DOCS_NOENT)
-	$(_xml2po) -m $(_xml2po_mode) -e -o $@ $^
+	$(_matexml2po) -m $(_matexml2po_mode) -e -o $@ $^
 
 
 ################################################################################
@@ -409,9 +409,9 @@ clean-doc-lc:
 	done
 #	.xml2.po.mo cleaning is obsolete as of 0.18.1 and could be removed in 0.20.x
 	@for lc in C $(_DOC_REAL_LINGUAS); do \
-	  if test -f "$$lc/.xml2po.mo"; then \
-	    echo "rm -f $$lc/.xml2po.mo"; \
-	    rm -f "$$lc/.xml2po.mo"; \
+	  if test -f "$$lc/.matexml2po.mo"; then \
+	    echo "rm -f $$lc/.matexml2po.mo"; \
+	    rm -f "$$lc/.matexml2po.mo"; \
 	  fi; \
 	done
 clean-doc-dir: clean-doc-lc
diff -upr mate-doc-utils-1.2.1-orig/tools/mate-doc-utils.pc.in mate-doc-utils-1.2.1/tools/mate-doc-utils.pc.in
--- mate-doc-utils-1.2.1-orig/tools/mate-doc-utils.pc.in	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/tools/mate-doc-utils.pc.in	2012-03-01 21:55:29.000000000 +0100
@@ -10,8 +10,8 @@ db2xhtml=${xsltdir}/docbook/html/db2xhtm
 db2omf=${xsltdir}/docbook/omf/db2omf.xsl
 mal2html=${xsltdir}/mallard/html/mal2html.xsl
 mal2xhtml=${xsltdir}/mallard/html/mal2xhtml.xsl
-malrnc=${xmldir}/mallard/1.0/mallard.rnc
-malrng=${xmldir}/mallard/1.0/mallard.rng
+malrnc=${xmldir}/mallard/1.0/matemallard.rnc
+malrng=${xmldir}/mallard/1.0/matemallard.rng
 
 Name: mate-doc-utils
 Description: MATE Documentation Utilities
diff -upr mate-doc-utils-1.2.1-orig/xml2po/Makefile.am mate-doc-utils-1.2.1/xml2po/Makefile.am
--- mate-doc-utils-1.2.1-orig/xml2po/Makefile.am	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/xml2po/Makefile.am	2012-03-02 01:22:51.000000000 +0100
@@ -1,24 +1,24 @@
 SUBDIRS = xml2po examples
 
 pkgconfigdir = $(datadir)/pkgconfig
-pkgconfig_DATA = xml2po.pc
+pkgconfig_DATA = matexml2po.pc
 
-man_MANS = xml2po.1
+man_MANS = matexml2po.1
 
 $(srcdir)/README: README.in
 	sed -e "s/\@VERSION\@/@VERSION@/g" < README.in > README
 
-xml2po.1: xml2po.1.xml
-	xsltproc --nonet $(srcdir)/xml2po.1.xml
+matexml2po.1: matexml2po.1.xml
+	xsltproc --nonet $(srcdir)/matexml2po.1.xml
 
 publish: dist
-	scp @PACKAGE@-@VERSION@.tar.gz danilo@kvota.net:/home/danilo/kvota.net/hacks/xml2po
+	scp @PACKAGE@-@VERSION@.tar.gz danilo@kvota.net:/home/danilo/kvota.net/hacks/matexml2po
 
 
 EXTRA_DIST = \
 	     README    \
 	     README.in \
 	     TODO      \
-	     xml2po.pc.in \
-	     xml2po.1.xml \
-	     xml2po.1
+	     matexml2po.pc.in \
+	     matexml2po.1.xml \
+	     matexml2po.1
diff -upr mate-doc-utils-1.2.1-orig/xml2po/xml2po/__init__.py mate-doc-utils-1.2.1/xml2po/xml2po/__init__.py
--- mate-doc-utils-1.2.1-orig/xml2po/xml2po/__init__.py	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/xml2po/xml2po/__init__.py	2012-03-01 22:19:22.000000000 +0100
@@ -2,20 +2,20 @@
 # Copyright (c) 2004, 2005, 2006 Danilo Šegan <danilo@gnome.org>.
 # Copyright (c) 2009 Claude Paroz <claude@2xlibre.net>.
 #
-# This file is part of xml2po.
+# This file is part of matexml2po.
 #
-# xml2po is free software; you can redistribute it and/or modify
+# matexml2po is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation; either version 2 of the License, or
 # (at your option) any later version.
 #
-# xml2po is distributed in the hope that it will be useful,
+# matexml2po is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with xml2po; if not, write to the Free Software Foundation, Inc.,
+# along with matexml2po; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 #
 import os
@@ -499,7 +499,7 @@ class XMLDocument(object):
             tmp.close()
         except:
             # We fail silently, and replace all entities if we cannot
-            # write .xml2po-entitychecking
+            # write .matexml2po-entitychecking
             # !!! This is not very nice thing to do, but I don't know if
             #     raising an exception is any better
             return False
@@ -559,11 +559,11 @@ class Main(object):
 
     def load_mode(self, modename):
         try:
-            module = __import__('xml2po.modes.%s' % modename, globals(), locals(), ['%sXmlMode' % modename])
+            module = __import__('matexml2po.modes.%s' % modename, globals(), locals(), ['%sXmlMode' % modename])
             return getattr(module, '%sXmlMode' % modename)
         except (ImportError, AttributeError):
             if modename == 'basic':
-                sys.stderr.write("Unable to find xml2po modes. Please check your xml2po installation.\n")
+                sys.stderr.write("Unable to find matexml2po modes. Please check your matexml2po installation.\n")
                 sys.exit(1)
             else:
                 sys.stderr.write("Unable to load mode '%s'. Falling back to 'basic' mode with automatic detection (-a).\n" % modename)
diff -upr mate-doc-utils-1.2.1-orig/xml2po/xml2po/Makefile.am mate-doc-utils-1.2.1/xml2po/xml2po/Makefile.am
--- mate-doc-utils-1.2.1-orig/xml2po/xml2po/Makefile.am	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/xml2po/xml2po/Makefile.am	2012-03-01 22:27:36.000000000 +0100
@@ -1,18 +1,18 @@
 SUBDIRS = modes
 
-xml2podir = $(pythondir)/xml2po
+xml2podir = $(pythondir)/matexml2po
 
-nodist_bin_SCRIPTS = xml2po
-CLEANFILES = xml2po
+nodist_bin_SCRIPTS = matexml2po
+CLEANFILES = matexml2po
 
-xml2po: xml2po.py.in
+matexml2po: matexml2po.py.in
 	sed -e "s/^VERSION =.*/VERSION = \"@VERSION@\"/" \
 	    -e "s+^#!.*python.*+#!$(PYTHON)+" \
-	  < $(srcdir)/xml2po.py.in > xml2po
-	chmod +x xml2po
+	  < $(srcdir)/matexml2po.py.in > matexml2po
+	chmod +x matexml2po
 
 xml2po_PYTHON = \
 	__init__.py
 
 EXTRA_DIST = \
-	xml2po.py.in
+	matexml2po.py.in
diff -upr mate-doc-utils-1.2.1-orig/xml2po/xml2po/modes/basic.py mate-doc-utils-1.2.1/xml2po/xml2po/modes/basic.py
--- mate-doc-utils-1.2.1-orig/xml2po/xml2po/modes/basic.py	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/xml2po/xml2po/modes/basic.py	2012-03-01 22:21:16.000000000 +0100
@@ -1,19 +1,19 @@
 # Copyright (c) 2004, 2005, 2006 Danilo Segan <danilo@gnome.org>.
 #
-# This file is part of xml2po.
+# This file is part of matexml2po.
 #
-# xml2po is free software; you can redistribute it and/or modify
+# matexml2po is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation; either version 2 of the License, or
 # (at your option) any later version.
 #
-# xml2po is distributed in the hope that it will be useful,
+# matexml2po is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with xml2po; if not, write to the Free Software Foundation, Inc.,
+# along with matexml2po; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 #
 
diff -upr mate-doc-utils-1.2.1-orig/xml2po/xml2po/modes/docbook.py mate-doc-utils-1.2.1/xml2po/xml2po/modes/docbook.py
--- mate-doc-utils-1.2.1-orig/xml2po/xml2po/modes/docbook.py	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/xml2po/xml2po/modes/docbook.py	2012-03-01 22:21:35.000000000 +0100
@@ -1,20 +1,20 @@
 # -*- coding: utf-8 -*-
 # Copyright (c) 2004, 2005, 2006 Danilo Segan <danilo@gnome.org>.
 #
-# This file is part of xml2po.
+# This file is part of matexml2po.
 #
-# xml2po is free software; you can redistribute it and/or modify
+# matexml2po is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation; either version 2 of the License, or
 # (at your option) any later version.
 #
-# xml2po is distributed in the hope that it will be useful,
+# matexml2po is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with xml2po; if not, write to the Free Software Foundation, Inc.,
+# along with matexml2po; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 #
 
diff -upr mate-doc-utils-1.2.1-orig/xml2po/xml2po/modes/gs.py mate-doc-utils-1.2.1/xml2po/xml2po/modes/gs.py
--- mate-doc-utils-1.2.1-orig/xml2po/xml2po/modes/gs.py	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/xml2po/xml2po/modes/gs.py	2012-03-01 22:21:50.000000000 +0100
@@ -1,19 +1,19 @@
 # Copyright (c) 2004, 2005, 2006 Danilo Segan <danilo@gnome.org>.
 #
-# This file is part of xml2po.
+# This file is part of matexml2po.
 #
-# xml2po is free software; you can redistribute it and/or modify
+# matexml2po is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation; either version 2 of the License, or
 # (at your option) any later version.
 #
-# xml2po is distributed in the hope that it will be useful,
+# matexml2po is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with xml2po; if not, write to the Free Software Foundation, Inc.,
+# along with matexml2po; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 #
 
diff -upr mate-doc-utils-1.2.1-orig/xml2po/xml2po/modes/Makefile.am mate-doc-utils-1.2.1/xml2po/xml2po/modes/Makefile.am
--- mate-doc-utils-1.2.1-orig/xml2po/xml2po/modes/Makefile.am	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/xml2po/xml2po/modes/Makefile.am	2012-03-01 22:27:21.000000000 +0100
@@ -1,3 +1,3 @@
-modesdir = $(pythondir)/xml2po/modes
+modesdir = $(pythondir)/matexml2po/modes
 modes_PYTHON = __init__.py basic.py docbook.py gs.py mallard.py ubuntu.py xhtml.py
 
diff -upr mate-doc-utils-1.2.1-orig/xml2po/xml2po/modes/xhtml.py mate-doc-utils-1.2.1/xml2po/xml2po/modes/xhtml.py
--- mate-doc-utils-1.2.1-orig/xml2po/xml2po/modes/xhtml.py	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/xml2po/xml2po/modes/xhtml.py	2012-03-01 22:22:25.000000000 +0100
@@ -1,20 +1,20 @@
 # Copyright (c) 2004, 2005, 2006 Danilo Segan <danilo@gnome.org>.
 # Copyright (c) 2006, 2009 Claude Paroz <claude@2xlibre.net>.
 #
-# This file is part of xml2po.
+# This file is part of matexml2po.
 #
-# xml2po is free software; you can redistribute it and/or modify
+# matexml2po is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation; either version 2 of the License, or
 # (at your option) any later version.
 #
-# xml2po is distributed in the hope that it will be useful,
+# matexml2po is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with xml2po; if not, write to the Free Software Foundation, Inc.,
+# along with matexml2po; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 #
 
diff -upr mate-doc-utils-1.2.1-orig/xml2po/xml2po/xml2po.py.in mate-doc-utils-1.2.1/xml2po/xml2po/xml2po.py.in
--- mate-doc-utils-1.2.1-orig/xml2po/xml2po/xml2po.py.in	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/xml2po/xml2po/xml2po.py.in	2012-03-01 22:18:52.000000000 +0100
@@ -3,24 +3,24 @@
 # Copyright (c) 2004, 2005, 2006 Danilo Šegan <danilo@gnome.org>.
 # Copyright (c) 2009 Claude Paroz <claude@2xlibre.net>.
 #
-# This file is part of xml2po.
+# This file is part of matexml2po.
 #
-# xml2po is free software; you can redistribute it and/or modify
+# matexml2po is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation; either version 2 of the License, or
 # (at your option) any later version.
 #
-# xml2po is distributed in the hope that it will be useful,
+# matexml2po is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with xml2po; if not, write to the Free Software Foundation, Inc.,
+# along with matexml2po; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 #
 
-# xml2po -- translate XML documents
+# matexml2po -- translate XML documents
 VERSION = "1.0.5"
 
 # Versioning system (I use this for a long time, so lets explain it to
@@ -59,7 +59,7 @@ OPTIONS may be some of:
     -l    --language=LANG      Set language of the translation to LANG
           --mark-untranslated  Set 'xml:lang="C"' on untranslated tags
 
-    -v    --version            Output version of the xml2po program
+    -v    --version            Output version of the matexml2po program
 
     -h    --help               Output this message
 
@@ -85,7 +85,7 @@ def main(argv):
         print >> sys.stderr, 'Running from source folder, modifying PYTHONPATH'
         sys.path.insert(0, name)
 
-    from xml2po import Main
+    from matexml2po import Main
 
     # Default parameters
     default_mode = 'docbook'
@@ -157,7 +157,7 @@ def main(argv):
         filenames.append(remaining_args.pop())
 
     try:
-        xml2po_main = Main(default_mode, operation, output, options)
+        matexml2po_main = Main(default_mode, operation, output, options)
     except IOError:
         print >> sys.stderr, "Error: cannot open file %s for writing." % (output)
         sys.exit(5)
@@ -171,17 +171,17 @@ def main(argv):
             print >> sys.stderr, "Error: You must specify MO file when merging translations."
             sys.exit(3)
 
-        xml2po_main.merge(mofile, filenames[0])
+        matexml2po_main.merge(mofile, filenames[0])
 
     elif operation == 'update':
-        xml2po_main.update(filenames, po_to_update)
+        matexml2po_main.update(filenames, po_to_update)
 
     elif origxml:
-        xml2po_main.reuse(origxml, filenames[0])
+        matexml2po_main.reuse(origxml, filenames[0])
 
     else:
         # Standard POT producing
-        xml2po_main.to_pot(filenames)
+        matexml2po_main.to_pot(filenames)
 
     if mofile_tmppath:
         os.remove(mofile_tmppath)
diff -upr mate-doc-utils-1.2.1-orig/xml2po/xml2po.1 mate-doc-utils-1.2.1/xml2po/xml2po.1
--- mate-doc-utils-1.2.1-orig/xml2po/xml2po.1	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/xml2po/xml2po.1	2012-03-01 21:08:01.000000000 +0100
@@ -19,18 +19,18 @@
 .\" * MAIN CONTENT STARTS HERE *
 .\" -----------------------------------------------------------------
 .SH "NAME"
-xml2po \- program to create a PO\-template file from a DocBook XML file and merge it back into a (translated) XML file
+matexml2po \- program to create a PO\-template file from a DocBook XML file and merge it back into a (translated) XML file
 .SH "SYNOPSIS"
-.HP \w'\fBxml2po\fR\ 'u
-\fBxml2po\fR [\fBOPTIONS\fR] [\fIXMLFILE\fR]
+.HP \w'\fBmatexml2po\fR\ 'u
+\fBmatexml2po\fR [\fBOPTIONS\fR] [\fIXMLFILE\fR]
 .SH "DESCRIPTION"
 .PP
 This manual page documents briefly the
-\fBxml2po\fR
+\fBmatexml2po\fR
 command\&.
 .PP
 
-\fBxml2po\fR
+\fBmatexml2po\fR
 is a simple Python program which extracts translatable content from free\-form XML documents and outputs gettext compatible POT files\&. Translated PO files can be turned into XML output again\&.
 .PP
 It can work it\'s magic with most "simple" tags, and for complicated tags one has to provide a list of all tags which are "final" (that will be put into one "message" in PO file), "ignored" (skipped over) and "space preserving"\&.
@@ -118,7 +118,7 @@ chapter2\&.xml
 .RS 4
 .\}
 .nf
-				\fB/usr/bin/xml2po\fR \fB\-o\fR book\&.pot book\&.xml chapter1\&.xml chapter2\&.xml
+				\fB/usr/bin/matexml2po\fR \fB\-o\fR book\&.pot book\&.xml chapter1\&.xml chapter2\&.xml
 			
 .fi
 .if n \{\
@@ -133,7 +133,7 @@ option:
 .RS 4
 .\}
 .nf
-				\fB/usr/bin/xml2po\fR \fB\-e \-o\fR book\&.pot book\&.xml
+				\fB/usr/bin/matexml2po\fR \fB\-e \-o\fR book\&.pot book\&.xml
 			
 .fi
 .if n \{\
@@ -152,9 +152,9 @@ option for each XML file:
 .RS 4
 .\}
 .nf
-				\fB/usr/bin/xml2po\fR \fB\-p\fR LANG\&.po \fB\-o\fR book\&.LANG\&.xml book\&.xml
-				\fB/usr/bin/xml2po\fR \fB\-p\fR LANG\&.po \fB\-o\fR chapter1\&.LANG\&.xml chapter1\&.xml
-				\fB/usr/bin/xml2po\fR \fB\-p\fR LANG\&.po \fB\-o\fR chapter2\&.LANG\&.xml chapter2\&.xml
+				\fB/usr/bin/matexml2po\fR \fB\-p\fR LANG\&.po \fB\-o\fR book\&.LANG\&.xml book\&.xml
+				\fB/usr/bin/matexml2po\fR \fB\-p\fR LANG\&.po \fB\-o\fR chapter1\&.LANG\&.xml chapter1\&.xml
+				\fB/usr/bin/matexml2po\fR \fB\-p\fR LANG\&.po \fB\-o\fR chapter2\&.LANG\&.xml chapter2\&.xml
 			
 .fi
 .if n \{\
@@ -169,7 +169,7 @@ option to expand entities, you should us
 .RS 4
 .\}
 .nf
-				\fB/usr/bin/xml2po\fR \fB\-e \-p\fR LANG\&.po \fB\-o\fR book\&.LANG\&.xml book\&.xml
+				\fB/usr/bin/matexml2po\fR \fB\-e \-p\fR LANG\&.po \fB\-o\fR book\&.LANG\&.xml book\&.xml
 			
 .fi
 .if n \{\
@@ -199,7 +199,7 @@ tmp\&.po
 to
 LANG\&.po
 and update your translation\&. Alternatively,
-\fBxml2po\fR
+\fBmatexml2po\fR
 provides the
 \fB\-u\fR
 option, which does exactly these two steps for you\&. The advantage is, that it also runs
@@ -212,7 +212,7 @@ if you decided earlier to expand entitie
 .RS 4
 .\}
 .nf
-				\fB/usr/bin/xml2po\fR \fB\-u\fR LANG\&.po book\&.xml
+				\fB/usr/bin/matexml2po\fR \fB\-u\fR LANG\&.po book\&.xml
 			
 .fi
 .if n \{\
diff -upr mate-doc-utils-1.2.1-orig/xml2po/xml2po.1.xml mate-doc-utils-1.2.1/xml2po/xml2po.1.xml
--- mate-doc-utils-1.2.1-orig/xml2po/xml2po.1.xml	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/xml2po/xml2po.1.xml	2012-03-01 20:35:22.000000000 +0100
@@ -11,7 +11,7 @@
   <!ENTITY dhemail     "<email>daniel.leidert@wgdd.de</email>">
   <!ENTITY dhusername  "Daniel Leidert">
   <!ENTITY dhucpackage "<refentrytitle>XML2PO</refentrytitle>">
-  <!ENTITY dhpackage   "xml2po">
+  <!ENTITY dhpackage   "matexml2po">
 ]>
 
 <refentry>
@@ -232,13 +232,13 @@
 				and <filename>chapter2.xml</filename> (external entities), run:
 			</para>
 			<programlisting>
-				<command>/usr/bin/xml2po</command> <option>-o</option> book.pot book.xml chapter1.xml chapter2.xml
+				<command>/usr/bin/matexml2po</command> <option>-o</option> book.pot book.xml chapter1.xml chapter2.xml
 			</programlisting>
 			<para>
 				To expand entities use the <option>-e</option> option:
 			</para>
 			<programlisting>
-				<command>/usr/bin/xml2po</command> <option>-e -o</option> book.pot book.xml
+				<command>/usr/bin/matexml2po</command> <option>-e -o</option> book.pot book.xml
 			</programlisting>
 		</refsect2>
 		<refsect2>
@@ -249,16 +249,16 @@
 				using <option>-p</option> option for each XML file:
 			</para>
 			<programlisting>
-				<command>/usr/bin/xml2po</command> <option>-p</option> LANG.po <option>-o</option> book.LANG.xml book.xml
-				<command>/usr/bin/xml2po</command> <option>-p</option> LANG.po <option>-o</option> chapter1.LANG.xml chapter1.xml
-				<command>/usr/bin/xml2po</command> <option>-p</option> LANG.po <option>-o</option> chapter2.LANG.xml chapter2.xml
+				<command>/usr/bin/matexml2po</command> <option>-p</option> LANG.po <option>-o</option> book.LANG.xml book.xml
+				<command>/usr/bin/matexml2po</command> <option>-p</option> LANG.po <option>-o</option> chapter1.LANG.xml chapter1.xml
+				<command>/usr/bin/matexml2po</command> <option>-p</option> LANG.po <option>-o</option> chapter2.LANG.xml chapter2.xml
 			</programlisting>
 			<para>
 				If you used the <option>-e</option> option to expand entities,
 				you should use it again to merge back the translation into an XML file:
 			</para>
 			<programlisting>
-				<command>/usr/bin/xml2po</command> <option>-e -p</option> LANG.po <option>-o</option> book.LANG.xml book.xml
+				<command>/usr/bin/matexml2po</command> <option>-e -p</option> LANG.po <option>-o</option> book.LANG.xml book.xml
 			</programlisting>
 		</refsect2>
 		<refsect2>
@@ -276,7 +276,7 @@
 			</programlisting>
 			<para>
 				Now rename <filename>tmp.po</filename> to <filename>LANG.po</filename> and
-				update your translation. Alternatively, <command>xml2po</command>
+				update your translation. Alternatively, <command>matexml2po</command>
 				provides the <option>-u</option> option, which does exactly these two steps for
 				you. The advantage is, that it also runs <command>msgfmt</command> to give you
 				a statistical output of translation status (count of translated, untranslated 
@@ -284,7 +284,7 @@
 				earlier to expand entities:
 			</para>
 			<programlisting>
-				<command>/usr/bin/xml2po</command> <option>-u</option> LANG.po book.xml
+				<command>/usr/bin/matexml2po</command> <option>-u</option> LANG.po book.xml
 			</programlisting>
 		</refsect2>
 	</refsect1>
diff -upr mate-doc-utils-1.2.1-orig/xml2po/xml2po.pc.in mate-doc-utils-1.2.1/xml2po/xml2po.pc.in
--- mate-doc-utils-1.2.1-orig/xml2po/xml2po.pc.in	2012-03-01 18:26:12.000000000 +0100
+++ mate-doc-utils-1.2.1/xml2po/xml2po.pc.in	2012-03-01 21:09:00.000000000 +0100
@@ -1,9 +1,9 @@
 prefix=@prefix@
 exec_prefix=${prefix}
 datarootdir=@datarootdir@
-modesdir=@pythondir@/xml2po
+modesdir=@pythondir@/matexml2po
 
-Name: xml2po
+Name: matexml2po
 Description: Tool for translating XML documents
 Version: @VERSION@
 Requires: libxml-2.0
