diff -upr caja-1.1.2-orig/src/caja-navigation-window-pane.c caja-1.1.2/src/caja-navigation-window-pane.c
--- caja-1.1.2-orig/src/caja-navigation-window-pane.c	2012-01-17 11:53:22.000000000 +0100
+++ caja-1.1.2/src/caja-navigation-window-pane.c	2012-03-02 07:29:26.000000000 +0100
@@ -649,12 +649,32 @@ caja_navigation_window_pane_always_use_l
     }
 }
 
+static gboolean
+location_button_should_be_active (CajaNavigationWindowPane *window)
+{
+	return eel_preferences_get_boolean (CAJA_PREFERENCES_ALWAYS_USE_LOCATION_ENTRY);
+}
+
+static void
+location_button_toggled_cb (GtkToggleButton *toggle,
+			    CajaNavigationWindowPane *pane)
+{
+	gboolean is_active;
+
+	is_active = gtk_toggle_button_get_active (toggle);
+	eel_preferences_set_boolean (CAJA_PREFERENCES_ALWAYS_USE_LOCATION_ENTRY, is_active);
+
+	if (is_active) {
+		caja_navigation_bar_activate (CAJA_NAVIGATION_BAR (pane->navigation_bar));
+	}
+}
+
 void
 caja_navigation_window_pane_setup (CajaNavigationWindowPane *pane)
 {
     GtkWidget *hbox;
     CajaEntry *entry;
-    GtkSizeGroup *header_size_group;
+    /* GtkSizeGroup *header_size_group; */
 
     pane->widget = gtk_vbox_new (FALSE, 0);
 
@@ -665,10 +685,33 @@ caja_navigation_window_pane_setup (CajaN
                         FALSE, FALSE, 0);
     gtk_widget_show (hbox);
 
-    header_size_group = CAJA_NAVIGATION_WINDOW (CAJA_WINDOW_PANE (pane)->window)->details->header_size_group;
+{
+		GtkWidget *image;
+		GtkWidget *button;
+
+		image = gtk_image_new_from_stock (GTK_STOCK_EDIT, GTK_ICON_SIZE_BUTTON);
+		gtk_widget_show (image);
+
+		button = g_object_new (GTK_TYPE_TOGGLE_BUTTON,
+				       "image", image,
+				       "focus-on-click", FALSE,
+				       "active", location_button_should_be_active (pane),
+				       NULL);
+
+		gtk_widget_set_tooltip_text (button,
+					     _("Toggle between button and text-based location bar"));
+
+		g_signal_connect (button, "toggled",
+				  G_CALLBACK (location_button_toggled_cb), pane);
+		pane->always_use_location_entry_toggle_button =  button;
+		gtk_box_pack_start (GTK_BOX (hbox), pane->always_use_location_entry_toggle_button, FALSE, FALSE, 0);
+		gtk_widget_show (pane->always_use_location_entry_toggle_button);
+	}
+
+	/* header_size_group = CAJA_NAVIGATION_WINDOW (CAJA_WINDOW_PANE (pane)->window)->details->header_size_group; */
 
     pane->path_bar = g_object_new (CAJA_TYPE_PATH_BAR, NULL);
-    gtk_size_group_add_widget (header_size_group, pane->path_bar);
+    /* gtk_size_group_add_widget (header_size_group, pane->path_bar); */
     gtk_widget_show (pane->path_bar);
 
     g_signal_connect_object (pane->path_bar, "path_clicked",
@@ -681,7 +724,7 @@ caja_navigation_window_pane_setup (CajaN
                         TRUE, TRUE, 0);
 
     pane->navigation_bar = caja_location_bar_new (pane);
-    gtk_size_group_add_widget (header_size_group, pane->navigation_bar);
+    /* gtk_size_group_add_widget (header_size_group, pane->search_bar); */
     g_signal_connect_object (pane->navigation_bar, "location_changed",
                              G_CALLBACK (navigation_bar_location_changed_callback), pane, 0);
     g_signal_connect_object (pane->navigation_bar, "cancel",
@@ -695,7 +738,7 @@ caja_navigation_window_pane_setup (CajaN
                         TRUE, TRUE, 0);
 
     pane->search_bar = caja_search_bar_new ();
-    gtk_size_group_add_widget (header_size_group, pane->search_bar);
+    /* gtk_size_group_add_widget (header_size_group, pane->search_bar); */
     g_signal_connect_object (pane->search_bar, "activate",
                              G_CALLBACK (search_bar_activate_callback), pane, 0);
     g_signal_connect_object (pane->search_bar, "cancel",
diff -upr caja-1.1.2-orig/src/caja-navigation-window-pane.h caja-1.1.2/src/caja-navigation-window-pane.h
--- caja-1.1.2-orig/src/caja-navigation-window-pane.h	2012-01-17 11:53:22.000000000 +0100
+++ caja-1.1.2/src/caja-navigation-window-pane.h	2012-03-02 07:20:21.000000000 +0100
@@ -50,6 +50,7 @@ struct _CajaNavigationWindowPane
     GtkWidget *widget;
 
     /* location bar */
+    GtkWidget *always_use_location_entry_toggle_button;
     GtkWidget *location_bar;
     GtkWidget *navigation_bar;
     GtkWidget *path_bar;
